lib/inputstreamhelper/__init__.py

```diff
@@ -274,13 +274,21 @@ def _update_widevine(self):
                 return
 
         wv_config = load_widevine_config()
-        latest_version = latest_widevine_version()
-        if 'x86' in arch():
+        if not wv_config:
+            log(3, 'Widevine config missing. Could not determine current version, forcing update.')
+            current_version = '0'
+        elif 'x86' in arch():
             component = 'Widevine CDM'
             current_version = wv_config['version']
         else:
             component = 'Chrome OS'
             current_version = select_best_chromeos_image(wv_config)['version']
+
+        latest_version = latest_widevine_version()
+        if not latest_version:
+            log(3, 'Updating widevine failed. Could not determine latest version.')
```

Comment:
I wouldn't state "Updating widevine failed" because that sounds rather fatal. I would go for something like `Skipping widevine update because latest version could no be determined.` Or even more specific if we know why determining latest version failed.

Reply:
Well, it's just what it is. update_widevine was called, but couldn't update. That's not skipping an update, that's an update failing.